lms/djangoapps/instructor/static/instructor/ProblemBrowser/data/reducers/index.js

```diff
@@ -0,0 +1,54 @@
+import {combineReducers} from 'redux';
+import {courseBlocksActions} from '../actions/constants';
+
+const BLOCK_TYPES = ['course', 'chapter', 'sequential', 'vertical', 'problem'];
```

Comment:
People may want to report on types other than `problem`. Ideally in the future, this would be "all structural XBlocks (course, chapter, sequential, vertical), and all XBlocks that implement `generate_report_data()`."

Since we can't do it that way yet, it might be better to do this as a blacklist, excluding `html`, `video`, and `discussion` which are non-reportable, and leaving everything else in by default.

Reply:
I've make the bock types that can be filtered by this component configurable via a parameter so it's not hard coded in the component anymore. I feel this makes the component more reusable in places where such a feature might be needed but with different or no filters.